### PR TITLE
update arista_eos_show_inventory

### PR DESCRIPTION
**Pull request by yshu95**
##### ISSUE TYPE
 - Bugfix Pull Request

##### COMPONENT
arista_eos_show_inventory

##### SUMMARY
I couldn't parse of arista_eos_show_inventory.
I think that, in netmiko at arista_eos, output of "show inventory" has space on end of line.
e.g.) netmiko send_command result
```
 'System has 4 fan modules\n'
 '  Module  Number of Fans  Model            Serial Number    \n'
 '  ------- --------------- ---------------- ---------------- \n'
 '  1       1               FAN-7000-R       N/A              \n'
 '  2       1               FAN-7000-R       N/A              \n'
 '  3       1               FAN-7000-R       N/A              \n'
 '  4       1               FAN-7000-R       N/A              \n'
```
So I updated arista_eos_show_inventory.textfsm.

And i think that recent "show inventory" command output has "Epoch" section.
e.g.)
```
 '  HW Version  Serial Number  Mfg Date   Epoch \n'
 '  ----------- -------------- ---------- ----- \n'
 '  11.12       XXXXXXXXXX    2017-04-21 01.00 \n'
```
So I updated arista_eos_show_inventory.raw.

test
```
tests/test_structured_data_against_parsed_reference_files.py::test_raw_data_against_mock[tests/arista_eos/show_inventory/arista_eos_show_inventory.raw] PASSED [  2%]
```

I'm sorry in my poor English.

### CODE REVIEW

File: templates/arista_eos_show_inventory.textfsm [hunk 1]
```diff
@@ -11,26 +11,26 @@ Chassis
...
-  ^\s+${VID}\s+${SN}\s+\d+-\d+-\d+$$ -> Record
+  ^\s+${VID}\s+${SN}\s+\d+-\d+-\d+\s+\d+\.\d+\s*$$ -> Record
   ^\s+${NAME}?\s+${DESCR}$$
   ^\s*System.+(power supply|power-supply) -> Power_Supply
 
 
 Power_Supply
   ^\s+Slot
   ^\s+-
-  ^\s+${PORT}\s+${NAME}\s+${SN}$$ -> Record
```
Comment (FragmentedPacket) [inline templates/arista_eos_show_inventory.textfsm:22]: What if you just got rid of the end of line requirement?

e.g. removing the `$$` at the end of each of these?

Comment (yshu95) [reply templates/arista_eos_show_inventory.textfsm:22]: Thank you for your reply.

As you said, it was possible to just erase the $$ end of the line. 
e.g.   ^\s+${PORT}?\s+\d+?\s+${NAME}?\s+${SN} -> Record

Should I fix it?

Comment (FragmentedPacket) [reply templates/arista_eos_show_inventory.textfsm:22]: I believe so and revert the existing test files and then add your own example to make sure all use cases are covered.

**Comment by yshu95** [comment]
Well, I fixed the original test and added a new example.
But the test does not pass because https://github.com/networktocode/ntc-templates/issues/794.

**Comment by yshu95** [comment]
Well, I fixed the original test and added a new example.

**Comment by jmcgill298** [comment]
Thanks @yshu95

**Pull request merged**